src/Core/Prefab/GlobeView.js

```diff
@@ -205,26 +206,9 @@ function GlobeView(viewerDiv, coordCarto, options) {
     const positionTargetCamera = positionCamera.clone();
```

Comment:
This could be moved in the `if (!options.noControls) {` block I think.

And maybe you should always set the camera position to `coordCarto` - even if the user didn't ask for `GlobeControls` to be used?